.github/workflows/clang_tidy.yml

```
@@ -0,0 +1,54 @@
+name: 🧹 clang-tidy
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.head_ref }}-ubuntu
```

The CI got cancelled because this collides with he ubuntu CI workflows:
```suggestion
  group: ${{ github.ref }}-${{ github.head_ref }}-clangtidy
```